docs/pages/application-access/jwt/grafana.mdx

```diff
@@ -0,0 +1,116 @@
+---
+title: Application Access with Grafana
+description: How to configure Teleport Application Access for Grafana.
+---
+
+This guide will help you configure Grafana [JWT
+authentication](https://grafana.com/docs/grafana/latest/setup-grafana/configure-security/configure-authentication/jwt/)
+with Teleport Application Access.
+
+## Prerequisites
+
+- Teleport cluster version >= `5.0.0` with running Auth/Proxy Services or
```

Comment:
I'm not sure we need a minimum version warning that old. If we do, we can use the standard callout:

```
<Details
  title="Version warning"
  opened={true}
  scope={["oss", "enterprise"]}
  scopeOnly={true}
  min="5.0"
>
 Blah
</Details>

```

And we can standardize the prereq for a running cluster with the partial:

```
(!docs/pages/includes/edition-prereqs-tabs.mdx!)
```